modules/oe_webtools_page_feedback/src/Form/WebtoolsPageFeedbackSettingsForm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_page_feedback\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides configuration form for the Page Feedback Form webtools widget.
+ */
+class WebtoolsPageFeedbackSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'oe_webtools_page_feedback_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('oe_webtools_page_feedback.settings');
+
+    $form['enabled'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enabled'),
+      '#description' => $this->t('Check this box if you would like to enable the Page feedback form on all the content pages.'),
```

Comment:
I was wrong here, let's say only: "Check this box if you would like to enable the Page feedback form on this site."